debian-tui/go编译单文件/install-tui-lanrenbao.sh

```
@@ -0,0 +1,102 @@
+    "buttons": [
+      {"id":"copy","actions":{"short":"none","hold_3s":"none","hold_9s":"none","hold_15s":"none"}},
+      {"id":"network","actions":{"short":"none","hold_3s":"none","hold_9s":"none","hold_15s":"none"}},
+      {"id":"rear_reset","actions":{"short":"none","hold_3s":"none","hold_9s":"none","hold_15s":"none"}}
+    ]
+  }
+}
+JSON
+chmod 600 "$ETC/config.json"
+
+log "安装 tank TUI 前端"
+[ -f "$SCRIPT_DIR/tank" ] && install -m 0755 "$SCRIPT_DIR/tank" /usr/local/bin/tank \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)